### ISSUE TITLE
Real package composability

### ISSUE THREAD

**Issue by RothAndrew** [defenseunicorns/zarf#212]
I'd like the ability to import other zarf configs so that I can compose a package from multiple sources. Here's an example of how it could look for pulling Flux into the Big Bang Core example

```yaml
kind: ZarfPackageConfig
metadata:
  name: big-bang-core-example
  description: "The Biggiest and Bangiest!"

components:
  - name: flux
    required: true
    include:
      source: "../flux/zarf.yaml"
      component: "flux"

  - name: kubescape
  ...
```

**Comment by RothAndrew** [comment]
Blocked by #208

**Comment by jeff-mccoy** [comment]
Closed in favor of #208.
